pkg/apis/meta/v1/time.go

```diff
@@ -83,6 +83,16 @@ func (t Time) Equal(u Time) bool {
 	return t.Time.Equal(u.Time)
 }
 
+// BeforeMicro reports whether the time instant t is before micro instance u.
```

Comment:
Same comment as other file, delete these or make them round correctly to micros.